### ISSUE TITLE
Resistor colour bands

### ISSUE THREAD

**Issue by drewnoakes** [KirillOsenkov/QuickInfo#23]
A friend added a feature like this to duckduckgo some time ago.

https://duckduckgo.com/?q=16+ohms&t=hg&ia=answer

You put in the resistance you want, in ohms, and it prints out the colour bands you would expect to see on a resistor.

![image](https://user-images.githubusercontent.com/350947/36938594-ac78fb10-1f1b-11e8-8015-5fd0f51095f2.png)

![image](https://user-images.githubusercontent.com/350947/36938587-a05378e2-1f1b-11e8-94a8-06ee27f99818.png)

**Comment by KirillOsenkov** [comment]
Yup, this would be nice. I don’t know anything about electronics myself, but if a good quality PR came along I’d likely take it.
